### ISSUE TITLE
Can Betterfox added to Mullvad browser?

### ISSUE THREAD

**Issue by dentaku65** [mullvad/mullvad-browser#384]
I use Betterfox mainly for perfomance in Firefox.
Is it worth integrating into Mullvad browser or does Mullvad browser already have its own optimization similar to Betterfox?
[](https://github.com/yokoffing/Betterfox)

**Comment by Thorin-Oakenpants** [comment]
No. Tor Browser and Mullvad Browser have reproducible builds, so can't using build optimizations (IANAE - [LTO](https://en.wikipedia.org/wiki/Interprocedural_optimization), [PGO](https://en.wikipedia.org/wiki/Profile-guided_optimization)), and that equates to about a 35-to-50% perf loss on speedometer 3 compared to Firefox ESR. It gets worse with the security slider set at safer, as JIT and ION are disabled.

No sort of "mostly snake oil magical pref tweaking" is gong to make a difference. Defaults are what they are for reasons - stability and not anecdoatal, i.e the best settings for a wide range of users/devices. In Firefox, feel free to change what works _for you_ on _your device_ ... not saying some tweaks don;t work for some people. But TB/MB is not the place for them

@ruihildt +1 close
